ARCHITECTURE.md

```diff
@@ -24,15 +24,9 @@ Presto aims to accomplish the above goals for users by creating a broad, powerfu
 3. **Composability**: Presto is designed for lakes and lakehouses. As such, it is engineered to fit cohesively in an ecosystem of other tools, and strives to integrate with industry standard specifications and libraries that help accomplish the user goals above.
 4. **Coding excellence**: Presto strives to maintain a high bar for contributions through good design, good abstractions, rigorous tests, and quality documentation.
 
-## Presto Community
+### Presto Community
```

Comment:
```suggestion
## Presto Community
```

I would keep it at H2 (parallel with the technical architecture)

Reply:
Fixed! Edit in new commit, squashed and pushed.